torch/csrc/cuda/expand_utils.cpp

```diff
@@ -0,0 +1,188 @@
+#include "torch/csrc/cuda/THCP.h"
```

Comment:
Why do we need these files (e.g. the same for csrc below)? Why can't we just call `THTensor_(new)`?